src/components/PotlukkHomePage.js

```diff
@@ -69,35 +69,35 @@ export default function PotlukkHomePage(){
             }
         }
         if(user.uId === btn.hostID) {
-            return(<button class="action-btn" onClick={() => action(true, btn.hostID)}>edit</button>)
+            return(<button className="action-btn" onClick={() => action(true, btn.hostID)}>edit</button>)
         } else {
-            return(<button class="action-btn" onClick={() => action(false, btn.hostID)}>attend</button>)
+            return(<button className="action-btn" onClick={() => action(false, btn.hostID)}>attend</button>)
         }
     }
 
     function EditPotlukk(btn) {
         const btnIndex = btn.index
         return(
-        <div class="potlukk-popup" id="edit-potlukk-popup">
+        <div className="potlukk-popup" id="edit-potlukk-popup">
         <ItemDisplayTable value={potlukks[btnIndex].id} />
         <button onClick={() => setShowEdit(false)}>Cancel</button>
         </div>)
     }
 
     function AttendPotlukk(btn) {
         const btnIndex = btn.index
-        return(<div class="potlukk-popup" id="attend-potlukk-popup">
+        return(<td className="potlukk-popup" id="attend-potlukk-popup">
```

Comment:
Console was complaining. This just gets rid of the warning/error

Reply:
I was wondering what the difference between the two were. Turns out class is a reserved keyword in JS and since React uses JSX, they decided to use className